.golangci.toml

```diff
@@ -1,45 +1,22 @@
-[issues]
-exclude-files = ["data/.*\\.gen\\.go", "data/generic_nullable_vector\\.go", "data/generic_vector\\.go"]
-
-[linters-settings.goconst]
-min-len = 5
-min-occurrences = 5
-
-[linters-settings.revive]
-ignore-generated-header = false
-
-[linters-settings.misspell]
-ignore-words = ["unknwon"]
-
-[linters-settings.depguard.rules.main]
-allow = [] # allow all
-deny = [
-  { pkg = "io/ioutil", desc = "Deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details." },
-]
+version = "2"
 
 [linters]
-disable-all = true
 enable = [
   "bodyclose",
-  "depguard",
```

Comment:
I have removed the "depguard" linter rule because the "allow all" mechanism no longer works so we'd need to specify all the packages we have. Also, there are no longer references to io/ioutil, which was the reason for adding it.